VadimVildanov/HW5/Task 2.py

```diff
@@ -0,0 +1,12 @@
+number_fibonacci = int(input("Enter your limit: "))
```

Comment:
good job 
but look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')